### ISSUE TITLE
Access to all returned data of dataTable query on server

### ISSUE THREAD

**Issue by alr2413** [yajra/laravel-datatables#1752]
### Summary of problem or feature request

First, thanks for this awesome package.
I want to calculate the sum of one columns of filtered data on my dataTable and send it to the client as an extra parameters. To make it more clear, assume that the selected column is price of an item and the extra parameter is the summation of price of all available items that is changed based on the requested search parameters.
As mentioned in dataTable docs, we can pass more extra parameters by the "with" method to the client. My problem is that how we can access all data on server.
Also note that this task can not be done in client-side because for a specific search parameters, only a portion of data are returned to the client based on the predefined pagination and the extra parameter (in my case "total price") is fixed for these search parameters. 
Thanks.

### System details

- Operating System: macOS High Sierra
- PHP Version: 7.1.10
- Laravel Version: 5.6.20
- Laravel-Datatables Version: 8.6.0

**Comment by yajra** [comment]
There is a less document api that you can use to get the filtered query. See https://github.com/yajra/laravel-datatables/blob/8.0/CHANGELOG.md#added and https://github.com/yajra/laravel-datatables/issues/1068 for ref.

```php
$dataTable = Datatables::of(App\User::query());
$response  = $dataTable->toJson();
$total = $dataTable-> getFilteredQuery()->sum('total');
// TODO: append queried total to JsonResponse.
return $response;
```

**Comment by alr2413** [comment]
Thanks a lot.

**Comment by yajra** [comment]
Hey, I just released a better way of doing this on #1758. Kindly give it a try. See also [docs](https://yajrabox.com/docs/laravel-datatables/master/response-with) for some ref. Thanks!
